test/batch_test.cc

```diff
@@ -0,0 +1,38 @@
+#include "benchmark/benchmark.h"
+
+#include <algorithm>
+#include <cassert>
+
+static void BM_batch_iteration_count(benchmark::State& state) {
+  const size_t batch_size = state.range(0);
+  size_t actual_iterations = 0;
+  while (state.KeepRunningBatch(batch_size)) {
+    actual_iterations += batch_size;
+  }
+  assert(state.iterations() >= state.max_iterations);
```

Comment:
you might need `src/check.h` or `../src/check.h`.

See https://github.com/google/benchmark/blob/4cfe790a253d2cb47f33b29676c0a27c462704cb/test/output_test_helper.cc#L7